docs/content/functioneel/zaaktypen_api.md

```diff
@@ -0,0 +1,188 @@
+koppelvlakdocumentatie]
+
+| **Aspect**      | **Beschrijving**                                                                                                                                                                                  |
+|-----------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Doel            | Het kunnen aanspreken van een voorziening voor het onderhouden en raadplegen van zaaktypecatalogus (ZTC) inclusief daarinopgenomen zaaktypen, statustypen, roltypen en informatieobjecttypen, rollen en relaties naar betrokkenen, objecten, zaaktypen en statustypen. |
+| Domein          | Zaakgericht werken                                                                                                                                                                                |
+| Provider        | [Zaaktypecataloguscomponent](https://www.gemmaonline.nl/index.php/Omgevingswet/1.5/id-3ef9cdd9-631c-4d3e-88c3-f756423d6314) (GEMMA2)                                                              |
+| Consumer        | Componenten waarmee zaken behandeld worden (Zaakafhandelcomponenten; ZAC’s) en (andere) componenten die zaaktypegegevens raadplegen.                                                              |
+| Informatiemodel | [IMZTC, versie 2.1](https://www.gemmaonline.nl/index.php/Informatiemodel_Zaaktypen_(ImZTC)) (in-ontwikkeling)                                         |
+| Specificaties   | <https://ref.tst.vng.cloud/zrc/api/v1/schema>                                                                                                                                                     |
+| Bijzonderheden  |  Deze API omvat (vooralsnog) de resources Catalogussen, Zaaktypen, Informatieobjecttypen, Roltypen, Statustypen en Zaakobjecten                                                    |
+
+Deze API omvat (vooralsnog) de resources Zaken, Klantcontacten, Rollen, Statussen en Eigenschappen.
+
+Op dit moment biedt deze API alleen nog de mogelijkheid de zaaktypecatalogus te raadplegen middels GET.
+
+### Resource: catalogus
+
+| **Aspect**     | **Beschrijving**                                                                                                                                                                 |
+|----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Aanleiding     | Een consumer wil een lijst van alle beschikbare CATALOGUSsen of een specifieke catalogus opvragen.         |
+| *Toevoegen*    |      |
+| - voorwaarde   | nvt.         |
+| - gevolg       | nvt.             |
+| *Wijzigen*     |      |
+| - voorwaarde   | nvt.         |
+| - gevolg       | nvt.             |
+| *Verwijderen*  |      |
```

Comment:
Catalogi kunnen net als Zaaktypen niet verwijderd worden. dit is dus echt 'n.v.t. (een Catalogus kan niet verwijderd worden, alleen beeindigd).' Dan kun je de rijen 'voorwaarde' en 'gevolg' weglaten.

Reply:
Is `Niet toegestaan` dan niet duidelijker om verwarring te voorkomen?

Reply:
Als we ZTC informatie willen beheren (toevoegen, bijwerken) vind ik eigenlijk dat we ook moeten kunnen verwijderen. Dit mag dan weliswaar alleen door de juiste persoon (beheerder oid) gedaan worden maar dat moet dan afgevangen worden met de juiste autorisatie en validaties. Datzelfde geldt voor zaken en documenten. Of komt er een aparte API voor het beheer van de gegevens?

Ik denk dan even aan het geval dat bijvoorbeeld een zaak vernietigd moet worden. Op één of andere wijze moet dit gecontroleerd kunnen gebeuren. Moet dit dan heel low-level op de database gedaan worden? Voor mijn gevoel moet er een API zijn om in zo'n geval te kunnen verwijderen.

De ontwerpkeuze "dat er van verwijderen geen sprake kan zijn" is naar mijn mening te kort door de bocht en niet correct. Een gewone zaakgerichte consumer als een generieke of specifieke Zaakafhandelcomponent (ZAC) kan niet verwijderen maar een archiefregistratiecomponent oid zou dat wel moeten kunnen.

Reply:
Scherp! Uiteindelijk moet alles verwijderd kunnen worden maar dan onder zeer geconditioneerde omstandigheden. Zo mag een zaaktype pas fysiek verwijderd worden als alle daarop gebaseerde zaken vanwege archiefregime zijn vernietigd dan wel 'overgebracht'. Ik stel voor om het verwijderen van wat dan ook vooralsnog buiten scope te plaatsen. Eerst maar 's een userstory die daar over gaat.

Reply:
Heren, dit is een nuttige discussie die in een eigen user-story gevoerd moet worden :-) De scope van deze functionele documentatie is op dit moment al breed genoeg, het lijkt me handig om te focussen op de huidige stand van zaken.

Reply:
Dan pakken we dit op bij een userstory en sluit ik deze conversatie